docs/dashboard_api.prod.yaml

```diff
@@ -508,6 +513,39 @@ paths:
           content: {}
       security:
       - api_key: []
+  /data_packages/{id}:
```

Comment:
@vlad-ignatov FYI - this should match your request, but now's a good time to spot check